docs/develop/getting-started/quick_start_docker.md

```diff
@@ -99,6 +100,64 @@ To learn more about how to run JavaScript apps in WasmEdge.
 - [AI inference application examples](https://github.com/second-state/wasmedge-quickjs/tree/main/example_js/tensorflow_lite_demo)
 - [Web service client examples with fetch()](https://github.com/second-state/wasmedge-quickjs/blob/main/example_js/wasi_http_fetch.js)
 
+## WasmEdge DockerSlim
+
+The `wasmedge/slim:{version}` Docker images provide a slim WasmEdge images built with [DockerSlim](https://dockersl.im) every releases.
+
+- Image `wasmedge/slim-runtime:{version}` includes only WasmEdge runtime with `wasmedge` command.
+- Image `wasmedge/slim:{version}` includes the following command line utilities:
+  - `wasmedge`
+  - `wasmedge compile`
+- Image `wasmedge/slim-tf:{version}` includes the following command line utilities:
+  - `wasmedge`
+  - `wasmedge compile`
+  - `wasmedge-tensorflow-lite`
+  - `wasmedge-tensorflow`
+  - `show-tflite-tensor`
+- The working directory of the release docker image is `/app`.
+
+<!-- prettier-ignore -->
+:::note
+The usage of `wasmedgec` is equal to `wasmedge compile`. We decide to deprecate `wasmedgec` in the future.
+:::
+
+### Examples
+
+Use `wasmedge compile` and `wasmedge` ([link](https://github.com/WasmEdge/WasmEdge/tree/master/examples/wasm)):
+
+```bash
+$ docker pull wasmedge/slim:{{ wasmedge_version }}
+
+$ docker run -it --rm -v $PWD:/app wasmedge/slim:{{ wasmedge_version }} wasmedge compile hello.wasm hello.aot.wasm
+[2022-07-07 08:15:49.154] [info] compile start
```

Comment:
Move this part to https://wasmedge.org/docs/category/deploy-wasmedge-apps-in-kubernetes